utils/common/src/main/java/org/apache/brooklyn/util/http/executor/UsernamePassword.java

```diff
@@ -16,28 +16,14 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.brooklyn.util.http.executor;
-
-import com.google.common.annotations.Beta;
 
-@Beta
-public class UsernamePassword implements Credentials {
-    private final String username;
-    private final String password;
+package org.apache.brooklyn.util.http.executor;
 
+/**
+ * @deprecated since 1.0.0 use {@link org.apache.brooklyn.util.http.auth.UsernamePassword} instead.
+ */
+public class UsernamePassword extends org.apache.brooklyn.util.http.auth.UsernamePassword {
```

Comment:
Should this also implement `org.apache.brooklyn.util.http.executor.Credentials`, for backwards compatibility?